### ISSUE TITLE
Phoenix instrumenter blows up on 404s.

### ISSUE THREAD

**Issue by electricshaman** [LeToteTeam/metrix#5]
https://github.com/LeToteTeam/dc/issues/470

**Comment by keathley** [comment]
I actually don't understand why the phoenix controller instrumenter would be sending signals when the plug conn doesn't actually have a phoenix controller in it. Like...are we doing something wrong? Are we supposed to have some fallback controller for this?

**Comment by GregMefford** [comment]
Yeah I thought there was a thing in the `v0.2.0` release of `Metrix` that solved this, but it turned out to not work.

**Comment by keathley** [comment]
I'm just very confused as to why phoenix would give us "controller instrumentation" when there's no controller.
